src/courseware/course/Course.jsx

```diff
@@ -53,19 +52,6 @@ const Course = ({
   const shouldDisplayTriggers = windowWidth >= breakpoints.small.minWidth;
   const daysPerWeek = course?.courseGoals?.selectedGoal?.daysPerWeek;
 
-  // Responsive breakpoints for showing the notification button/tray
```

Comment:
[inform]: There are a number of elements in the Palm version of MFE Learning that do not work correctly, which we will not fix as part of the scope for improving accessibility. This code and some tests associated with it have been removed.